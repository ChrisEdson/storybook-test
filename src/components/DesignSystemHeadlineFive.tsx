import { StyleProp, Text, TextStyle } from 'react-native';

import { styles } from './DesignSystemHeadlineFive.style';

type Props = {
  children: React.ReactNode;
  style?: StyleProp<Pick<TextStyle, 'color' | 'textAlign'>>;
};

export const HeadlineFive = ({ children, style }: Props): JSX.Element => (
  <Text style={[styles.text, style]}>{children}</Text>
);
