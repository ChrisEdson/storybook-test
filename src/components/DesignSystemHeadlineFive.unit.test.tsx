import "react";
import { render } from "@testing-library/react-native";

import { HeadlineFive } from "./DesignSystemHeadlineFive";

describe("DesignSystemHeadlineFive", () => {
  it("renders text correctly", () => {
    expect(
      render(<HeadlineFive>Test</HeadlineFive>).toJSON()
    ).toMatchSnapshot();
  });
});
