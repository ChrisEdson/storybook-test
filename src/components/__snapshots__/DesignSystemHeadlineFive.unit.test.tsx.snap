// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DesignSystemHeadlineFive renders text correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#042D49",
        "fontFamily": "TTFirsNeue-Medium",
        "fontSize": 14,
        "letterSpacing": -0.22,
        "lineHeight": 18,
      },
      undefined,
    ]
  }
>
  Test
</Text>
`;
